### ISSUE TITLE
Branding Assets Integration

### ISSUE THREAD

**Issue by KrashLeviathan** [hammer-io/yggdrasil#104]
Dependent upon #103 : Branding Asset Creation.

Take the newly-created branding assets and integrate them into the website (and elsewhere as needed). Off the top of my head, here are some of the places:

- [x] Yggdrasil frontend logo, favicon, colors, typography, etc
- [x] Email template logo, colors, typography
- [x] Public API documentation on GitHub (add logos)
- [x] Backend documentation generated by `apidoc`
- [x] Team website
- [x] Update READMEs for Skadi, Tyr, Yggdrasil

This will happen across multiple projects, so reference this issue in each PR but don't close it until they're all complete.

**Comment by KrashLeviathan** [comment]
Created hammer-io/endor#90
Created hammer-io/hammer-io.github.io#9
Won't fix READMEs
